demo/App.vue

```diff
@@ -14,60 +14,42 @@
 </template>
 
 <script>
-import moment from "moment";
-import "../dist/vuetify-daterange-picker.css";
-import DateRange from "../";
+import subDays from 'date-fns/sub_days';
+import format from 'date-fns/format';
```

Comment:
I think both of these lines can be merged to say..
```js
import { subDays, format } from 'date-fns'
```
right?

Reply:
Yes in the case of `date-fns` we can merge them I guess. Some libraries are compatible with tree-shaking.  I am very cautious about final bundle sizes so I usually import what I use.